src/styles/supportModal.css

```diff
@@ -9,7 +9,11 @@
 }
 
 .coz-support-modal .coz-support-modal-wrapper {
-  margin-top: 1.4rem;
+  margin-top: 1rem;
```

Comment:
We have some classes in UI that can help you to remove this one. 

You should use `u-mt-1` and remove this new classe ?